### PR TITLE
Pro change support

### PR DESCRIPTION
**Pull request by albertkol**
## Done

- Make changes to /support for Ubuntu pro release

## QA

- Go to https://ubuntu-com-12078.demos.haus/support 
- Check against the copy: https://docs.google.com/document/d/1AU2dwTXpQk2UuEvSuAqRKrbPWYb-LR0ZGvWu6xw95I8/edit#

## Issue / Card

Fixes #[Issue](https://github.com/canonical/web-squad/issues/6024)

### CODE REVIEW

File: templates/support/index.html [hunk 1]
```diff
@@ -274,13 +277,15 @@ <h2>Apps support</h2>
         <tr>
           <th>&nbsp;</th>
           <th>&nbsp;</th>
-          <th class="u-align--center"colspan="2"><h4>Ubuntu advantage</h4></th>
+          <th class="u-align--center"colspan="2"><h4>Ubuntu Pro</h4></th>
```
Comment (sowasred2012) [inline templates/support/index.html:280]: The copy doc makes it look like this should be deleted. If it should stay, it should be:

`<h3 class="p-heading--4">Ubuntu Pro</h3>`

And I don't think it should be contained within a table row

File: templates/support/index.html [hunk 2]
```diff
@@ -10,10 +10,10 @@
 <section class="p-strip--suru is-dark">
   <div class="row u-equal-height">
     <div class="col-8">
-      <h1>Ubuntu Advantage</h1>
+      <h1>Ubuntu Pro</h1>
       <h4>Security, support and managed open source for enterprises</h4>
```
Comment (sowasred2012) [inline templates/support/index.html:14]:
```suggestion
      <h2 class="p-heading--4">Security, support and managed open source for enterprises</h2>
```

File: templates/support/index.html [hunk 3]
```diff
@@ -297,6 +302,16 @@ <h2>Apps support</h2>
               <img src="https://res.cloudinary.com/canonical/image/fetch/f_auto,q_auto,fl_sanitize,w_16,h_16/https://assets.ubuntu.com/v1/f1a7515d-tick-darkaubergine.svg">
             </div>
           </td>
+          <td class="u-align--center">
+            <div class="lazy-loaded">
+              <img src="https://res.cloudinary.com/canonical/image/fetch/f_auto,q_auto,fl_sanitize,w_16,h_16/https://assets.ubuntu.com/v1/f1a7515d-tick-darkaubergine.svg">
```
Comment (sowasred2012) [inline templates/support/index.html:307]: Every instance of this tick icon on this page needs to include alt text:
```suggestion
              <img src="https://res.cloudinary.com/canonical/image/fetch/f_auto,q_auto,fl_sanitize,w_16,h_16/https://assets.ubuntu.com/v1/f1a7515d-tick-darkaubergine.svg" alt="Included">
```

**Review by sowasred2012 — CHANGES_REQUESTED** [review]
Looking good, just a few accessibility tweaks needed

File: templates/support/index.html (hunk 2 above)
Comment (albertkol) [reply templates/support/index.html:14]: Deleted.

File: templates/support/index.html (hunk 1 above)
Comment (albertkol) [reply templates/support/index.html:280]: Deleted.

Comment (albertkol) [reply templates/support/index.html:280]: Waiting to get a confirmation if this is to be changed from a table or not..

Comment (albertkol) [reply templates/support/index.html:280]: Table format is fine. But had to add ROS too.

**Review by sowasred2012 — APPROVED** [review]
Nice, thanks!

**Pull request merged**